### PR TITLE
include project thoth kafkatopics to the kafka instance

### PR DESCRIPTION
**Pull request by harshad16**
include project thoth kafkatopics to the kafka instance
Signed-off-by: Harshad Reddy Nalla <hnalla@redhat.com>

Related-to: thoth-station/thoth-application#709

### CODE REVIEW

File: odh/overlays/moc/kafka/topics/thoth-missing-package.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+apiVersion: kafka.strimzi.io/v1beta1
+kind: KafkaTopic
+metadata:
+  name: cnv-prod.thoth.package-update.missing-package
```
Comment (HumairAK) [inline odh/overlays/moc/kafka/topics/thoth-missing-package.yaml:4]: this one is the same as the one [here](https://github.com/operate-first/apps/pull/66/files#diff-6bd89ca4910acd4a1d011b1ec905c2eff1b1ed487b53cead6154710625b24149R4) -- can you pick a different topic name?

Comment (harshad16) [reply odh/overlays/moc/kafka/topics/thoth-missing-package.yaml:4]: oh thanks for the catch.
will fix this.

Comment (harshad16) [reply odh/overlays/moc/kafka/topics/thoth-missing-package.yaml:4]: the one in the thoth-missing-package-version is fixed.

**Review by HumairAK — APPROVED** [review]

**Pull request merged**